### PR TITLE
Include initial blurb about Google-Native

### PR DESCRIPTION
**Pull request by viveklak**

### CODE REVIEW

File: themes/default/content/docs/intro/cloud-providers/gcp-native/_index.md [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+
+The GCP provider supports several options for providing access to Google Cloud credentials.  See [GCP setup page]({{< relref "setup" >}}) for details.
+
+## Getting Started
+
+GCP Native is currently in *public preview*. The quickest way to get started with GCP is to follow the steps described in the [README](https://github.com/pulumi/pulumi-gcp-native#readme).
+
+Some interesting examples are available complete with instructions:
+
+* [Google Cloud Functions](https://github.com/pulumi/examples/tree/master/gcp-ts-functions): Create a serverless function
```
Comment (viveklak) [inline themes/default/content/docs/intro/cloud-providers/gcp-native/_index.md:35]: TODO - point to GCP native versions of these.

File: themes/default/content/docs/intro/cloud-providers/gcp-native/_index.md [hunk 2]
```diff
@@ -0,0 +1,146 @@
...
+
+                return nil
+        })
+}
+```
+
+{{% /choosable %}}
+{{% choosable language csharp %}}
+
+```csharp
```
Comment (viveklak) [inline themes/default/content/docs/intro/cloud-providers/gcp-native/_index.md:115]: Need to replace with gcp-native version in csharp

File: themes/default/content/docs/intro/cloud-providers/_index.md [hunk 3]
```diff
@@ -18,6 +18,7 @@ aliases: ["/docs/reference/clouds/"]
 * [Azure-Native]({{< relref "./azure" >}})
 * [AzureAD]({{< relref "./azuread" >}})
 * [Google Cloud]({{< relref "./gcp" >}})
+* [Native Google Cloud (GCP-Native)]({{< relref "./gcp-native" >}})
```
Comment (mikhailshilkov) [inline themes/default/content/docs/intro/cloud-providers/_index.md:21]: Should it be just `GCP-Native` (similar to `Azure-Native`)? Or what's our prose here @infin8x

File: themes/default/content/docs/intro/cloud-providers/gcp-native/_index.md [hunk 4]
```diff
@@ -0,0 +1,146 @@
+---
+title: GCP-Native
+meta_desc: This page provides on an overview of the Native Google Cloud (GCP-Native) provider for Pulumi.
```
Comment (mikhailshilkov) [inline themes/default/content/docs/intro/cloud-providers/gcp-native/_index.md:3]:
```suggestion
meta_desc: This page provides on an overview of the native Google Cloud (GCP-Native) provider for Pulumi.
```

I think that's how we write "native Azure provider"

File: themes/default/content/docs/intro/cloud-providers/gcp-native/_index.md (hunk 1 above)
Comment (mikhailshilkov) [reply themes/default/content/docs/intro/cloud-providers/gcp-native/_index.md:35]: This one can be reused from the gcp-native repo

File: themes/default/content/docs/intro/cloud-providers/gcp-native/_index.md [hunk 5]
```diff
@@ -0,0 +1,146 @@
...
+[the Pulumi examples repo](https://github.com/pulumi/examples).
+
+## Libraries
+
+The following packages are available in package managers:
+
+* JavaScript/TypeScript: [`@pulumi/gcp-native`](https://www.npmjs.com/package/@pulumi/gcp-native)
+* Python: [`pulumi-gcp-native`](https://pypi.org/project/pulumi-gcp-native/)
+* Go: [`github.com/pulumi/pulumi-gcp-native/sdk/go/gcp`](https://github.com/pulumi/pulumi-gcp-native)
+* .NET: [`Pulumi.Gcp.Native`](https://www.nuget.org/packages/Pulumi.Gcp.Native)
```
Comment (mikhailshilkov) [inline themes/default/content/docs/intro/cloud-providers/gcp-native/_index.md:143]:
```suggestion
* .NET: [`Pulumi.GcpNative`](https://www.nuget.org/packages/Pulumi.GcpNative)
```

File: themes/default/content/docs/intro/cloud-providers/_index.md (hunk 3 above)
Comment (infin8x) [reply themes/default/content/docs/intro/cloud-providers/_index.md:21]: Just `Google-Native` works, I think

File: themes/default/content/docs/intro/cloud-providers/gcp-native/_index.md (hunk 4 above)
Comment (infin8x) [reply themes/default/content/docs/intro/cloud-providers/gcp-native/_index.md:3]: Yep, correct

Comment (infin8x) [reply themes/default/content/docs/intro/cloud-providers/gcp-native/_index.md:3]:
```suggestion
meta_desc: This page provides on an overview of the native Google Cloud (Google-Native) provider for Pulumi.
```

**Review by infin8x — COMMENTED** [review]
This looks great!

File: themes/default/content/docs/intro/cloud-providers/gcp-native/_index.md (hunk 4 above)
Comment (viveklak) [reply themes/default/content/docs/intro/cloud-providers/gcp-native/_index.md]: fixed.

File: themes/default/content/docs/intro/cloud-providers/gcp-native/_index.md (hunk 5 above)
Comment (viveklak) [reply themes/default/content/docs/intro/cloud-providers/gcp-native/_index.md]: fixed.

File: themes/default/content/docs/intro/cloud-providers/gcp-native/_index.md (hunk 1 above)
Comment (viveklak) [reply themes/default/content/docs/intro/cloud-providers/gcp-native/_index.md]: > This one can be reused from the gcp-native repo

See https://github.com/pulumi/examples/pull/967

File: themes/default/content/docs/intro/cloud-providers/google-native/_index.md [hunk 6]
```diff
@@ -0,0 +1,146 @@
...
+The Google Cloud provider supports several options for providing access to Google Cloud credentials.  See [setup page]({{< relref "setup" >}}) for details.
+
+## Getting Started
+
+Google-Native provider is currently in <strong> public preview </strong>. The quickest way to get started with Google Cloud is to follow the steps described in the [README](https://github.com/pulumi/pulumi-google-native#readme).
+
+Some interesting examples are available complete with instructions:
+
+* [Google Cloud Functions](https://github.com/pulumi/examples/tree/master/google-native-ts-functions): Create a serverless function
+* [Ruby on Rails on GKE and Google Cloud SQL](https://github.com/pulumi/examples/tree/master/google-native-ts-k8s-ruby-on-rails-postgresql): Containerized Ruby on Rails app using Google Cloud SQL PostgresSQL, GKE and Google container registry.
```
Comment (viveklak) [inline themes/default/content/docs/intro/cloud-providers/google-native/_index.md:36]: This is added in https://github.com/pulumi/examples/pull/966

File: themes/default/content/docs/intro/cloud-providers/google-native/_index.md [hunk 7]
```diff
@@ -0,0 +1,144 @@
+---
+title: Google-Native
+meta_desc: This page provides on an overview of the native Google Cloud (Google-Native) provider for Pulumi.
```
Comment (infin8x) [inline themes/default/content/docs/intro/cloud-providers/google-native/_index.md:3]:
```suggestion
meta_desc: This page provides on an overview of the native Google Cloud provider for Pulumi: Google-Native.
```

**Review by infin8x — APPROVED** [review]

**Review by infin8x — APPROVED** [review]
Reaffirming: this LGTM to me. I made some tweaks to the draft to match the new nomenclature we're standardizing around for the provider

**Review by mikhailshilkov — APPROVED** [review]
LGTM!

**Pull request merged**